README.md

```diff
@@ -4,15 +4,15 @@ Nuage MetroAE configuration template engine.
 
 Version 1.1.0
 
-This tool reads JSON or Yaml files of templates and user-data to
+This tool reads Excel spreadsheets (.xlsx files) or Yaml files of templates and user-data to
 write a configuration to a VSD or to revert (remove) said configuration.
 
 ## Overview
 
 MetroAE config is a Python-based engine which can apply configuration to VSDs
 via templates.  The templates provide an abstraction around the VSD
 configuration model simplifying and validating the required data.  Data is
-provided by the user in the form of Yaml or JSON files and is transformed and
+provided by the user in the form of Yaml or Excel spreadsheets (.xlsx files) files and is transformed and
```

Comment:
Move the "files" to after "Yaml": "... in the form of Yaml files or Excel spreadsheets (.xlsx files)...."